app/servers/main/logic/js/server.js

```diff
@@ -146,3 +146,10 @@ function savePage() {
 
 function evictPage() {
 }
+
+window.onunload =  function() {
+    frequencyDialer.cleanup();
+    eventsAPI.disconnect();
+    historyAPI.disconnect();
+    favoritesAPI.disconnect();
+};
```

Comment:
I think I still need it. Here, there are the clients that the `logic` server uses. So I need to close them when the server goes down (when it has 0 clients), to propagate the close to the history, favorites... servers. The Server object (the one instanciated with `new Server('logic)`) does not know about them.